Cargo.toml

```diff
@@ -30,18 +30,20 @@ llvm_8 = ["bcc-sys/llvm_8"]
 llvm_9 = ["bcc-sys/llvm_9"]
 static = ["bcc-sys/static"]
 specific = []
+usdt = []
```

Comment:
There may already be edge cases like this... but we do have a problem if we build with invalid combinations - eg ```cargo check --features "v0_4_0 usdt"``` creates a nasty mess of errors

Reply:
I guess I could actually just abandon the `usdt` feature flag and stick to versions, since the code is now organized under `core` in the same way?

Reply:
Though, to be fair, 0.4.0 isn't in CI and may have been broken already - better example is 0.6.0 + usdt. I'll probably open a new PR to drop some older versions...

Reply:
Yeah, there's definitely problems with 0.4.0 on its own, but the point you make when adding in the `usdt` feature flag is still valid.  It was a decent attempt, but I think my attempt flew in the face of "feature flags should always be additive" and if they can't be, then that's a good signal that they're being misused.

I reverted this back to a simple version feature flag-based approach, which fixes the issue.